armi/reactor/blueprints/tests/test_gridBlueprints.py

```diff
@@ -381,6 +381,12 @@ def test_simpleRead(self):
             self.assertEqual(gridDesign4.gridContents[-4, -3], "1")
 
     def test_simpleReadLatticeMap(self):
+        """Read lattice map and create a grid.
+
+        .. test:: Define a lattice map in reactor core
+            :id: T_ARMI_BP_GRID
+            :tests: R_ARMI_BP_GRID
```

Comment:
I like it.  I think there should ALSO be a `.. test::` crumb for this req on the `test_tiny_map()` test above.

(Though they will need slightly different test IDs, of course.)